File: SyncKit/Classes/CoreData/QSServerToken+CoreDataProperties.m
//
//  QSServerToken+CoreDataProperties.m
//  
//
//  Created by Manuel Entrena on 24/03/2018.
//
//

#import "QSServerToken+CoreDataProperties.h"

@implementation QSServerToken (CoreDataProperties)

+ (NSFetchRequest<QSServerToken *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"QSServerToken"];
}

@dynamic token;

@end

File: SyncKit/Classes/CoreData/QSServerToken+CoreDataProperties.swift
//
//  QSServerToken+CoreDataProperties.swift
//  
//
//  Created by Manuel Entrena on 02/06/2019.
//
//

import Foundation
import CoreData


extension QSServerToken {

    @nonobjc class func fetchRequest() -> NSFetchRequest<QSServerToken> {
        return NSFetchRequest<QSServerToken>(entityName: "QSServerToken")
    }

    @NSManaged var token: NSData?

}
